.flake8

```diff
@@ -0,0 +1,37 @@
+[flake8]
+ignore =
+    E203,
+    E241,
+    W291,
+    E302,
+    E501,
+    W503,
+    E741,
+
+per-file-ignores =
+    pyteal/compiler/optimizer/__init__.py:          F401
+    examples/application/asset.py:                  F403, F405
+    examples/application/security_token.py:         F403, F405
+    examples/application/vote.py:                   F403, F405
+    examples/signature/atomic_swap.py:              F403, F405
+    examples/signature/basic.py:                    F403, F405
+    examples/signature/dutch_auction.py:            F403, F405
+    examples/signature/periodic_payment_deploy.py:  F403, F405
+    examples/signature/recurring_swap.py:           F403, F405
+    examples/signature/split.py:                    F403, F405
+    examples/signature/periodic_payment.py:         F403, F405
+    examples/signature/recurring_swap_deploy.py:    F403, F405
+    pyteal/__init__.py:                             F401, F403
+    pyteal/ir/ops.py:                               E221
+    tests/module_test.py:                           F401, F403
+    # Temporary ignores while merging
+    pyteal/ast/abi/array_base_test.py:              F403, F405
+    pyteal/ast/abi/array_dynamic_test.py:           F403, F405
+    pyteal/ast/abi/array_static_test.py:            F403, F405
+    pyteal/ast/abi/bool_test.py:                    F403, F405
+    pyteal/ast/abi/method_return_test.py:           F403, F405
+    pyteal/ast/abi/tuple_test.py:                   F403, F405
+    pyteal/ast/abi/type_test.py:                    F403, F405
+    pyteal/ast/abi/uint_test.py:                    F403, F405
+    pyteal/ast/abi/util_test.py:                    F403, F405
```

Comment:
nit: you could have done `pyteal/ast/abi/*.py`

Reply:
@jasonpaulos Responding inline and resolving so that I can merge.  If you'd like to discuss further, feel welcomed to reopen.  Happy to continue discussion as is helpful.

> I just merged https://github.com/algorand/pyteal/pull/283 into master -- do you want to bundle those changes into this PR as well?

I suggest separating because I anticipate large diffs.

> you can use wildcards here, so pyteal/ast/abi/*.py would work

Agree that wildcards work here.  Originally, I implemented a single wildcard.  I switched because it makes incremental changes for the subsequent PR easier to validate.